recipes/kwant/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "kwant" %}
+{% set version = "1.1.2" %}
+{% set variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://gitlab.kwant-project.org/kwant/kwant/repository/archive.tar.gz?ref=v{{ version }}
+  sha256: a47544a8416b19a56da3d511610c9e09ed3bf84f66461d9f95f0f6fefebd0179
+
+build:
+  skip: true  # [win]
+  skip: true  # [py3k] 
```

Comment:
These can be combined `skip: true  # [win or py3k]`. Also is it that this version is Python 2 only or is it that this version is the last one that supports Python 2, but also supports Python 3?